plugin/storage/cassandra/schema/v003.cql.tmpl

```diff
@@ -201,4 +201,29 @@ CREATE TABLE IF NOT EXISTS ${keyspace}.dependencies_v2 (
         'max_threshold': '32',
         'class': 'org.apache.cassandra.db.compaction.SizeTieredCompactionStrategy'
     }
-    AND default_time_to_live = ${dependencies_ttl};
\ No newline at end of file
+    AND default_time_to_live = ${dependencies_ttl};
+
+-- adaptive sampling tables
```

Comment:
Doesn't it deserve a new file?

Reply:
Perhaps I could add more comments to explain why it's creating these tables? If we create a second file we will need changes to `jaeger-cassandra-schema` to optionally apply it. Perhaps changes to the operator as well?

I think the cost of the extra empty tables is low as long as we add some comments so people know why they exist?